README.md

```diff
@@ -1,3 +1,12 @@
+# gitlab
+
+[![Build Status](https://travis-ci.org/voxpupuli/puppet-gitlab.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-gitlab)
+[![Code Coverage](https://coveralls.io/repos/github/voxpupuli/puppet-gitlab/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-gitlab)
```

Comment:
This badge is only applicable to projects that have ruby code (ie a `lib` directory).